apps/api/src/hyp3_api/api-spec/openapi-spec.yml.j2

```diff
@@ -25,7 +25,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/costs"
+                $ref: "#/components/schemas/costs_response"
```

Comment:
Forgive my ignorance, but are we change the `/cost` endpoint to `costs_response` here?

Reply:
No, the URL endpoint is defined by `/costs` on line 19 above. This line references the `costs_response` schema defined on line 120 below.